Templates/CSharp/Model/ComplexType.cs.tt

```diff
@@ -26,14 +26,19 @@ if (!complex.IsAbstract && complex.Derived != null && (complex.Base == null || c
     attributeStringBuilder.Append("[JsonConverter(typeof(DerivedTypeConverter))]");
 }
 
+var namespaceName = complex.Namespace.GetNamespaceName();
+
 #>
 
-namespace <#=complex.Namespace.GetNamespaceName()#>
+namespace <#=namespaceName#>
 {
     using System;
     using System.Collections.Generic;
     using System.IO;
     using System.Runtime.Serialization;
+<# if (!this.IsGraphNamespace(namespaceName)) { #>
+    using Microsoft.Graph;
```

Comment:
That would work too, though it would be a much bigger change as we'd have to update every entity, base class, collectionpage, etc reference in each of the templates.